### DISCUSSION TITLE
PSA: 0.3mm layer height doesn't work well

### DISCUSSION THREAD

**Discussion by mythmon** [aderusha/DDD-Printable-Wall-Control-System#43] [General]
I initially printed these hooks with 0.3mm layer height, for faster printing times. This seemed to work well at first, but I had a really hard time press fitting the pieces together and inserting them into the wall control panels. After some debugging I realized that the pieces were just a little too thick.

The tab that goes through the wall control panel is designed at 2.2mm*. That's not evenly divisible by 0.3. When I sliced that with PrusaSlicer's 0.3mm layer height preset it consistently rounded up, increasing the overall size to 2.3mm or 2.4mm (the preset uses 0.2mm first layer). I attribute this to my difficulty assembling and using the parts I've printed.

\* Measured by taking the reverse engineered FreeCAD version, exporting it to STEP, and then measuring in Fusion 360

**Comment by root-hal9000** [comment]
Thanks for pointing this out! That explains some of the problems I had recently! I suppose that I should stick to 0.20mm or could use .40mm when using my 0.6mm nozzle

**Comment by root-hal9000** [comment]
PS.: variable layer height in PrusaSlicer can be your friend here - I set the profile to 0.2mm to get the tabs right, but make all the other layers where that doesn't matter larger
![image](https://github.com/aderusha/DDD-Printable-Wall-Control-System/assets/2352828/ea0d26d8-f68e-4cce-919f-ba317e22d882)
